embedded/sql/engine_test.go

```diff
@@ -6083,15 +6087,15 @@ func TestCopyCatalogToTx(t *testing.T) {
 	})
 
 	t.Run("indexing should work with new catalogue", func(t *testing.T) {
-		_, _, err := engine.Exec(ctx, nil, "INSERT INTO table1 (name, amount) VALUES ('name1', 10), ('name1', 10)", nil)
+		_, _, err := engine.Exec(context.Background(), nil, "INSERT INTO table1 (name, amount) VALUES ('name1', 10), ('name1', 10)", nil)
 		require.ErrorIs(t, err, store.ErrKeyAlreadyExists)
 
 		// should fail due non-available index
-		_, err = engine.Query(ctx, nil, "SELECT * FROM table1 ORDER BY amount DESC", nil)
+		_, err = engine.Query(context.Background(), nil, "SELECT * FROM table1 ORDER BY amount DESC", nil)
 		require.ErrorIs(t, err, ErrNoAvailableIndex)
 
 		// should use primary index by default
-		r, err := engine.Query(ctx, nil, "SELECT * FROM table1", nil)
+		r, err := engine.Query(context.Background(), nil, "SELECT * FROM table1", nil)
```

Comment:
I feel we could have made these changes in a different PR, just to not make this big